src/libraries/Microsoft.PowerFx.Core/Functions/IHasUnsupportedFunctions.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+namespace Microsoft.PowerFx.Core.Functions
+{
+    internal interface IHasUnsupportedFunctions
+    {
+        bool IsDeprecated { get; }
+
+        bool IsSupported { get; }
```

Comment:
>bool [](http://example.com/codeflow?start=8&length=4)

can this be `IsNotSupported`
Because anybody that doesn't implement this interface really has a default value of IsNotSupported=false